api/src/main/java/org/jboss/forge/roaster/model/util/Assert.java

```diff
@@ -30,11 +33,20 @@ public static void isFalse(boolean condition, String message)
       }
    }
 
-   public static void notNull(Object object, String message) throws IllegalStateException
+   public static void notNull(Object object, String message)
```

Comment:
How about we delete this class and use Objects.requireNonNull instead?

Reply:
What's about doing this in a new PR?

Reply:
Sure, sounds good to me